src/app/templates/app/license_requests.html

```diff
@@ -0,0 +1,51 @@
+{% extends 'app/base.html' %}
+{% load static %}
+{% load staticfiles %}
+{% block body1 %}
+
+<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.10.16/css/jquery.dataTables.css"> 
+
+<div id="messages" class="messages">
+</div>
+<p class ="lead"> {{ error }}</p>
+<div class="panel panel-default">
+<div class="panel-heading"> <p class="lead">License Requests List</p> </div>
+<div class="panel-body">
+    <table id="requests_table" class="display dataTable">
+      <thead>
+        <tr>
+          <th>Fullname</th>
+          <th>Short Identifier</th>
+          <th>Submission Date</th>
+        </tr>
+      </thead>
+      <tbody>
+        {% if licenseRequests %}
+        {% for elem in licenseRequests %}
+          <tr>
+            <td>{{elem.fullname}}</td>
+            <td>{{elem.shortIdentifier}}</td>
+            <td>{{elem.submissionDatetime}}</td>
+          </tr>
+        {% endfor %}
+        {% endif %}
+      </tbody>
+    </table>
+
+{% include "app/modal.html" %} 
+</div>
+{% endblock %}
+{% block script_block %}
+<script type="text/javascript">
+  $(document).ready(function () {
+      $("#licenserequestspage").addClass('linkactive');
+      });
+</script>
+
+<script type="text/javascript">
+  $(document).ready( function () {
+      $('#requests_table').DataTable();
```

Comment:
Currently the search happens over all the fields- Like Fullname, short identifier and Date. Maybe we can add more specific search later. I think this JS datatables provides that capability. 
You can add this to the ToDo list. No need to worry about it now.